quartz/bootstrap-cli.mjs

```diff
@@ -41,6 +41,23 @@ const CommonArgv = {
     default: false,
     describe: "print out extra logging information",
   },
+  "in-directory": {
+    string: true,
+    alias: ["i"],
+    describe: "what directory to copy/create symlink from",
+  },
+  setup: {
```

Comment:
rename to `strategy`

Reply:
what part should be renamed to strategy? both `setup` and `links` would be their respective strategy

Reply:
`setup`

Reply:
alias for `source` and `strategy` is (both `-s`) is conflicting, how should we handle this?

Reply:
`strategy` can be `-X` akin to how `git` differentiates

Reply:
if you're using an alias anyways imo it doesn't matter of it's slightly obscure lol